.vib/opencart/cypress/cypress/integration/opencart_spec.js

```diff
@@ -0,0 +1,114 @@
+/// <reference types="cypress" />
+import { random } from './utils';
+
+it('allows a user to register', () => {
+  cy.visit('/index.php?route=account/login');
+  cy.contains('Continue').click();
+  cy.fixture('users').then((user) => {
+    cy.get('#input-firstname').type(`${user.newUser.firstName}.${random}`);
+    cy.get('#input-lastname').type(`${user.newUser.lastName}.${random}`);
+    cy.get('#input-email').type(`${random}.${user.newUser.email}`);
+    cy.get('#input-telephone').type(user.newUser.phone);
+    cy.get('#input-password').type(`${user.newUser.password}.${random}`);
+    cy.get('#input-confirm').type(`${user.newUser.password}.${random}`);
+  });
+  cy.get('[type="checkbox"]').check();
+  cy.contains('Continue').click();
+  cy.contains('Your Account Has Been Created!');
+});
+
+it('allows a user to place an order', () => {
+  cy.visit('/');
+  cy.contains('Desktop').click();
+  cy.contains('Mac').click();
+  cy.contains('iMac').click();
+  cy.contains('Add to Cart').click();
+  cy.contains('.alert', 'Success:');
```

Comment:
Same as above, although maybe you need to add `:` here to differentiate it from another string present. If so, that's perfect 😄